devops/arangodb.yml

```
@@ -111,3 +95,19 @@
         name: arangodb3
         state: restarted
       become: true
+
+    - name: Template file to server
```

Instead of such a generic description, this should say what is exactly happening (remember this is what we end up seeing when we run provisioning). Maybe:

```suggestion
    - name: Install arangosh script for user provisioning to server
```